docs/content/en/api.md

```diff
@@ -28,12 +28,18 @@ All [Vue I18n properties and methods](http://kazupon.github.io/vue-i18n/api/#vue
 
   - **Arguments**:
     - locale: (type: `string`)
+    - forcePrefix: (type: `boolean`)
```

Comment:
I'd want to keep the changes as simple as possible while fixing your use case.

Is this new argument necessary? I have hard time imagining why would someone want the locale prefix and sometimes not.

Reply:
By adding a split for `{ routing: 'default', switchLocale: 'default' }`, I thought that in some situations we wouldn't want to globally change the behavior of the method to something else, so I decided to add this argument.

However, I now think that such a situation is actually hard to imagine.